tests/queryset/queryset.py

```diff
@@ -4635,17 +4635,18 @@ class LastLogin(EmbeddedDocument):
             ip = StringField()
 
         class User(Document):
-            id = ObjectIdField('_id')
+            id = StringField(primary_key=True)
```

Comment:
Could you elaborate on this? Was it an issue to keep `id = ObjectIdField('_id')`?

Reply:
I wanted to be able to pass the id, so I would know what will be the id for the later assertion checks. I guess I can query it right after the objects creation, I just don't think that is the point of the test.